includes/class-sensei-modules.php

```diff
@@ -1838,7 +1906,7 @@ public function setup_modules_taxonomy() {
 		$args = array(
 			'public'             => true,
 			'hierarchical'       => true,
-			'show_admin_column'  => true,
+			'show_admin_column'  => false,
```

Comment:
Because the modules taxonomy admin column had to be disabled, I had to add a custom column to the lessons as well.